src/node.cc

```diff
@@ -3224,6 +3224,21 @@ void SetupProcessObject(Environment* env,
                     Integer::New(env->isolate(),
                                  v8::kPromiseHandlerAddedAfterReject));
 
+  // process._debugOptions
+  Local<Object> debugOptions = Object::New(env->isolate());
+  READONLY_DONT_ENUM_PROPERTY(process,
+                              "_debugOptions",
```

Comment:
Everything in `src/node_constants.cc` looks like something defined at compile time, therefore it's no place for these. You're sure about that?